functions_and_scope_assignment.js

```diff
@@ -0,0 +1,283 @@
+//QUESTION 1
```

Comment:
There are some things that I left feedbacks on for few questions. Also try to revisit this assignment later and finish questions after 7. It is important to clearly understand how functions work and when to return values. Keep it up!